### PR TITLE
feat: Reveal tokens subcommand

### PR DESCRIPTION
**Pull request by jawnsy**
* Mark tokens subcommand as visible
* Re-generate Markdown documentation for the commands
* Add JSON output for "tokens ls" command, as with "envs ls"
* Add integration test cases

Tested the commands:

```console
[coder@jawnsy-m docs]$ coder tokens ls -o json
warning: version mismatch detected
  | Coder CLI version: unknown
  | Coder API version: 1.14.0+351-gb3b505259-20210124
  | 
  | tip: download the appropriate version here: https://github.com/cdr/coder-cli/releases
[{"id":"RQAv4Tvhiw","name":"abc","application":true,"user_id":"6006fa81-d49b8c0d52e920fe40f5ca90","last_used":"0001-01-01T00:00:00Z"},{"id":"rAvv33PmmY","name":"","application":false,"user_id":"6006fa81-d49b8c0d52e920fe40f5ca90","last_used":"2021-01-22T21:19:58.65074Z"},{"id":"XTCWEeOvoo","name":"","application":false,"user_id":"6006fa81-d49b8c0d52e920fe40f5ca90","last_used":"2021-01-25T00:13:34.62261Z"},{"id":"vl7pIPT8dX","name":"","application":false,"user_id":"6006fa81-d49b8c0d52e920fe40f5ca90","last_used":"2021-01-25T00:22:36.56265Z"}]
[coder@jawnsy-m docs]$ coder tokens create abcdef
warning: version mismatch detected
  | Coder CLI version: unknown
  | Coder API version: 1.14.0+351-gb3b505259-20210124
  | 
  | tip: download the appropriate version here: https://github.com/cdr/coder-cli/releases
Oo6zFqfK3t-tvUWLtWQPY5Qrj4s14E1Rq
[coder@jawnsy-m docs]$ coder tokens ls
warning: version mismatch detected
  | Coder CLI version: unknown
  | Coder API version: 1.14.0+351-gb3b505259-20210124
  | 
  | tip: download the appropriate version here: https://github.com/cdr/coder-cli/releases
ID            Name      Application    UserID                               LastUsed                               
Oo6zFqfK3t    abcdef    true           6006fa81-d49b8c0d52e920fe40f5ca90    0001-01-01 00:00:00 +0000 UTC          
RQAv4Tvhiw    abc       true           6006fa81-d49b8c0d52e920fe40f5ca90    0001-01-01 00:00:00 +0000 UTC          
rAvv33PmmY              false          6006fa81-d49b8c0d52e920fe40f5ca90    2021-01-22 21:19:58.65074 +0000 UTC    
XTCWEeOvoo              false          6006fa81-d49b8c0d52e920fe40f5ca90    2021-01-25 00:13:34.62261 +0000 UTC    
vl7pIPT8dX              false          6006fa81-d49b8c0d52e920fe40f5ca90    2021-01-25 00:22:36.56265 +0000 UTC    
[coder@jawnsy-m docs]$ coder tokens regen Oo6zFqfK3t
warning: version mismatch detected
  | Coder CLI version: unknown
  | Coder API version: 1.14.0+351-gb3b505259-20210124
  | 
  | tip: download the appropriate version here: https://github.com/cdr/coder-cli/releases
Oo6zFqfK3t-r9OHpN1h6BJPhnHWFIYbD3
[coder@jawnsy-m docs]$ coder tokens rm Oo6zFqfK3t
warning: version mismatch detected
  | Coder CLI version: unknown
  | Coder API version: 1.14.0+351-gb3b505259-20210124
  | 
  | tip: download the appropriate version here: https://github.com/cdr/coder-cli/releases
[coder@jawnsy-m docs]$ coder tokens ls
warning: version mismatch detected
  | Coder CLI version: unknown
  | Coder API version: 1.14.0+351-gb3b505259-20210124
  | 
  | tip: download the appropriate version here: https://github.com/cdr/coder-cli/releases
ID            Name    Application    UserID                               LastUsed                               
RQAv4Tvhiw    abc     true           6006fa81-d49b8c0d52e920fe40f5ca90    0001-01-01 00:00:00 +0000 UTC          
rAvv33PmmY            false          6006fa81-d49b8c0d52e920fe40f5ca90    2021-01-22 21:19:58.65074 +0000 UTC    
XTCWEeOvoo            false          6006fa81-d49b8c0d52e920fe40f5ca90    2021-01-25 00:13:34.62261 +0000 UTC    
vl7pIPT8dX            false          6006fa81-d49b8c0d52e920fe40f5ca90    2021-01-25 00:22:36.56265 +0000 UTC
```

### CODE REVIEW

File: internal/cmd/tokens.go [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
 	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
 )
 
 func tokensCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:    "tokens",
 		Short:  "manage Coder API tokens for the active user",
-		Hidden: true,
+		Hidden: false,
```
Comment (cmoog) [inline internal/cmd/tokens.go:20]:
```suggestion
```

**Review by cmoog — APPROVED** [review]

File: internal/cmd/tokens.go (hunk 1 above)
Comment (jawnsy) [reply internal/cmd/tokens.go]: Good call, fixed!

**Review by bpmct — APPROVED** [review]
nice!

**Comment by jawnsy** [comment]
@cmoog Something from the above output that I'm a bit unsure about, the tokens listed are partial (only the part before the hyphen), so even though regen changes the key (the latter part changes), the `coder tokens ls` output looks the same. I assume this is as designed but figured I should highlight it to you just in case.

**Review by tychoish — APPROVED** [review]

**Pull request merged**